alfredxml/type.go

```diff
@@ -152,6 +155,20 @@ type ClientNum struct {
 	N       int `xml:",chardata"`
 }
 
+// S2nDevice is used for xml coding
+type S2nDevice struct {
+	XMLName    xml.Name
+	Name       string  `xml:"name,omitempty"`
+	MacAddress string  `xml:"mac,omitempty"`
+	Firmware   string  `xml:"fw,omitempty"`
+	Hardware   string  `xml:"hw,omitempty"`
+	RemoteMac  string  `xml:"removemac,omitempty"`
+	Signal     int     `xml:"signal,omitempty"`
+	Capacity   float32 `xml:"capacity,omitempty"`
+	Latency    string  `xml:"latency,omitempty"`
```

Comment:
Could this be a more specific type like `time.Duration`, or is the format not well defined and better left as a `string`?